roles/openshift_storage_glusterfs/tasks/glusterblock_deploy.yml

```diff
@@ -64,3 +77,24 @@
   - "glusterblock_pod.results.results[0]['items'] | lib_utils_oo_collect(attribute='status.conditions') | lib_utils_oo_collect(attribute='status', filters={'type': 'Ready'}) | map('bool') | select | list | count == 1"
   delay: 10
   retries: "{{ (glusterfs_timeout | int / 10) | int }}"
+
+- name: Get cluster ID for GlusterBlock
+  command: "{{ glusterfs_heketi_client }} cluster list --json"
+  register: glusterblock_clusterid
+  changed_when: False
+
+- set_fact:
+    glusterblock_clusterid: "{{ (glusterblock_clusterid.stdout | from_json).clusters[0] }}"
+
+- name: Generate GlusterFS StorageClass file
+  template:
+    src: "{{ openshift.common.examples_content_version }}/gluster-block-storageclass.yml.j2"
+    dest: "{{ mktemp.stdout }}/gluster-block-storageclass.yml"
+
+- name: Create Gluster block StorageClass
+  oc_obj:
+    state: present
+    kind: storageclass
+    name: "glusterfs-{{ glusterfs_name }}-block"
+    files:
+    - "{{ mktemp.stdout }}/gluster-block-storageclass.yml"
```

Comment:
Move all the above to `glusterfs_common.yml` and place it after the include of glusterblock_deploy`. Also make it conditional on `openshift_storage_glusterfs_block_storageclass`.